mava/systems/sable/types.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Callable, Dict, Tuple
+
+from chex import Array, PRNGKey
+from flax.core.frozen_dict import FrozenDict
+from jumanji.types import TimeStep
+from optax._src.base import OptState
+from typing_extensions import NamedTuple
+
+
+class SableNetworkConfig(NamedTuple):
```

Comment:
I think this was a dataclass for MAT, but I could be wrong

Reply:
Yeah I think it was defined there as well but isn't weird to import MAT config then we need as well to import PPOTransition

Reply:
Since the motivation for this is just standardization I will edit MAT to have NamedTuple since it makes more sense to keep it NamedTuple